<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyVideoPlayer</class>
 <widget class="QWidget" name="MyVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="FFmpegWidget" name="playWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-40</y>
     <width>782</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>761</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_url">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_select">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pause">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FFmpegWidget</class>
   <extends>QWidget</extends>
   <header>ffmpeg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
